### ISSUE TITLE
Open updated blog post in new window

### ISSUE THREAD

**Issue by orschiro** [classeur/classeur#167]
Hi @benweet,

I think it would be very useful if you could open the updated blog post in a new window by clicking on the following notification. 

What do you think?

Warmly,

~Robert

![image](https://cloud.githubusercontent.com/assets/170947/17664299/5d1d1b2c-62f3-11e6-9a44-9802201a31fe.png)

**Comment by benweet** [comment]
Indeed, I think it's a useful feature to have and quite easy to add.
Cheers
